src/rdb/rdb.c

```diff
@@ -485,6 +487,16 @@ rdb_close(struct rdb_storage *storage)
 	D_FREE(db);
 }
 
+static bool
+rdb_get_use_leases(void)
+{
+	char   *name = "RDB_USE_LEASES";
```

Comment:
it's probably worth documenting this environment variable in daos/docs/admin/env_variables.md

Reply:
OK, documented. (I actually chose not to document them because I felt they required too much internal knowledge and were unlikely to be proactively used by administrators.)